sns_ik_lib/src/sns_vel_ik_base.cpp

```diff
@@ -294,7 +294,7 @@ SnsIkBase::ExitCode SnsVelIkBase::solve(const Eigen::MatrixXd& J, const Eigen::V
     return ExitCode::InternalError;
   }
   else if (*taskScaleCS < MINIMUM_FINITE_SCALE_FACTOR) {
-    ROS_WARN("Secondary goal is infeasible! scaling --> zero");
+    ROS_DEBUG_THROTTLE(0.5, "Secondary goal is infeasible! scaling --> zero");
```

Comment:
If you're already making this debug, do you still need to throttle it? same question for other similar change

Reply:
Right. I agree.

Reply:
Thanks for your comments! :)